install-pcan.md

```diff
@@ -0,0 +1,20 @@
+# Install PCAN-M.2 Driver for Linux (Ubuntu 16.04)
+
+Official page:  [link](https://www.peak-system.com/fileadmin/media/linux/index.htm)
+Manual:         [link](https://www.peak-system.com/fileadmin/media/linux/files/PCAN-Driver-Linux_UserMan_eng.pdf)
+
+```bash
+sudo apt-get install libpopt-dev libelf-dev
```

Comment:
BTW what did you need these packages for? Are those referenced in the official PCan manual?

Reply:
ASWR might be required by non-PCI modules. Ran on a clean Ubuntu installation, none was needed.